src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +23,34 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    @Transactional(readOnly = true)
+    public List<LineResponse> findAllLines() {
+        List<Line> lines = lineRepository.findAll();
+
+        return lines.stream()
+                .map(line -> LineResponse.of(line))
```

Comment:
```suggestion
                .map(LineResponse::of)
```

메소드 레퍼런스를 활용해볼 수 있겠네요~